packages/office-ui-fabric-react/src/components/Persona/PersonaPresence/PersonaPresence.base.tsx

```diff
@@ -72,30 +74,27 @@ export class PersonaPresenceBase extends BaseComponent<IPersonaPresenceProps, {}
   }
 
   private _onRenderIcon = (className?: string, style?: React.CSSProperties): JSX.Element => (
-    <Icon className={className} iconName={this._determineIcon()} style={style} />
+    <Icon className={className} iconName={determineIcon(this.props.presence, this.props.isOutOfOffice)} style={style} />
   );
+}
 
-  private _determineIcon = (): string | undefined => {
-    const { presence } = this.props;
+function determineIcon(presence: PersonaPresenceEnum | undefined, isOutOfOffice: boolean | undefined): string | undefined {
+  if (!presence) {
+    return undefined;
+  }
 
-    if (presence !== PersonaPresenceEnum.none) {
-      let userPresence = PersonaPresenceEnum[presence as PersonaPresenceEnum];
+  const oofIcon = 'SkypeArrow';
```

Comment:
>oofIcon [](start = 8, length = 7)

More of a comment for later, but would be nice with type safety of these icons

Reply:
The quoted text kind of confuses me, but yes I agree. 

Matt and I discussed possibly switching these icons out to be SVGs instead to get rid of the font rendering issues in different browsers. Perhaps this could be part of that work. :)